pkg/operator/resourcesynccontroller/resourcesynccontroller.go

```diff
@@ -10,6 +10,13 @@ import (
 	"github.com/openshift/cluster-kube-controller-manager-operator/pkg/operator/operatorclient"
 )
 
+func AddSyncCSRControllerCA(resourceSyncController *resourcesynccontroller.ResourceSyncController) error {
```

Comment:
Nit, I'd call this `SyncCSRControllerCA` you're not adding anything.

Reply:
It is adding new rule to `resourceSyncController` object underneath for syncing the ConfigMap :)
Given this has to be a non-member method I wanted to avoid calling it `SyncCSRControllerCA` which would imply this actually does the sync.